pkg/volume/volume.go

```diff
@@ -42,9 +51,10 @@ type Builder interface {
 	// directory path, which may or may not exist yet.  This may be called
 	// more than once, so implementations must be idempotent.
 	SetUpAt(dir string) error
-	// IsReadOnly is a flag that gives the builder's ReadOnly attribute.
-	// All persistent volumes have a private readOnly flag in their builders.
-	IsReadOnly() bool
+	// GetReadWriteMode returns the ReadWriteMode of the volume.
+	// If the mode is ReadWrite or ReadOnlyManaged, the Kubelet will perform
+	// ownership and selinux label management on appropriate volumes.
```

Comment:
nit: I wouldn't expect this interface to contain information about how the user of this interface `kubelet` will be using this information.